addons/website/static/src/js/widgets/theme.js

```diff
@@ -699,6 +719,14 @@ var ThemeCustomizeDialog = Dialog.extend({
             $dropdown.find('.o_theme_customize_dropdown_btn').remove();
             $dropdown.prepend($btn);
         });
+        _.each(this.$('.o_theme_customize_auto'), function (el) {
+            var $el = $(el);
+            var value = $el.data('value');
+            var isActive = value === parseInt(self.style.getPropertyValue('--' + $el.data('variable')));
+            $el.closest('.o_theme_customize_option').toggleClass('active', isActive);
+            $el.closest('.o_theme_customize_selection').siblings('button')
+                .toggleClass('o_theme_customize_option_font_' + value, isActive);
```

Comment:
After some thinking, this cannot work like that. This is relying on the fact the auto widgets are in a dropdown which is not ok. Try removing the `<selection/>` which wraps all fonts... and this will not work anymore while it should. You were right in your first investigation: they should not be all "checked".

Before your task, they were like 3 cases:
- We use xmlid on an unique option: this appears like a switch (like for "Show Sign In")
- We use xmlid on a set of options: this appears like multiple options where only one is marked selected (like for "Boxed" / "Full"
- We do not use xmlid but a widget: used for colors, logo font size -> the inner input is always marked as checked as we rely on custom JS (in processChange) to do a custom behavior.

Your widget is in fact a new case. It should work as if there was an xmlid even though this is a widget. Indeed: if you set a unique auto widget "Really Big Logo" (as in my example), you would want it to appear as a switch, which is *checked* or not. If you set a set of auto widgets (like your case with fonts), you want only one to be *checked* at a time. The condition we talked about in `_updateValues` is in fact the condition for the option to be checked in the first place. Try implementing that.

Reply:
Hey @qsm-odoo 
Could you check what I've done when you have some time ?

So, to give some words on what has been done:
First sorry for the delay, I really had to go deeper in the code understanding because I blocked on some stuff... So I took more time, I really do my best not to make bullsh*ts :) 

Concerning your explanation and what we talked about, this is what it looks like now (with some test inputs - for the title font - which have been removed from my code before pushing) : https://ibb.co/9c3sM0W. In fact, everything is working and is interconnected correctly so let's check my code now :) !

Plus, little word of explanation about the switch element (the one alone). If you untick the switch it comes back to default value. So for example on our screenshot, if you untick the switch, it will be Bootstrap font again. It seems logic but I'm clarifying here to make sure this is what we want.